reel-picker/src/main/java/com/innowisegroup/reelpicker/picker/ui/TimePickerFragment.kt

```diff
@@ -24,12 +24,12 @@ internal class TimePickerFragment : Fragment() {
     private var hours: CustomNumberPicker? = null
     private var minutes: CustomNumberPicker? = null
 
-    private var localDateTime: LocalDateTime? = null
-    private var minDateTime: LocalDateTime? = null
-    private var maxDateTime: LocalDateTime? = null
-    private var localTime: LocalTime? = null
-    private var minTime: LocalTime? = null
-    private var maxTime: LocalTime? = null
+    lateinit var localDateTime: LocalDateTime
+    private lateinit var minDateTime: LocalDateTime
+    private lateinit var maxDateTime: LocalDateTime
```

Comment:
Why do we need 
lateinit var localDateTime: LocalDateTime
    private lateinit var minDateTime: LocalDateTime
    private lateinit var maxDateTime: LocalDateTime

Reply:
Why not to pass the instanse of LocalTime in bundle arguments